app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
```

Comment:
`cash` stands for money in coins and notes, this name does not apply to this context. Replace this name with another, e.g. `cache_storage`.